applications_tests/gls_sharp_navier_stokes/flow_around_rbf.prm

```diff
@@ -75,8 +75,9 @@ subsection particles
   end
 
   subsection particle info 0
-    set type            = rbf
-    set shape arguments = rbf_test_shape.input
+    set type                       = rbf
+    set shape arguments            = rbf_test_shape.input
+    set mesh based precalculations = false
```

Comment:
```suggestion
    set mesh based precalculations = false
```
I think this would require an hyphen to be gramatically correct but we can let this pass